# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/prelink/pt_gnu_stack.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

 this definition is private for binutils, seems the author forgot that
 when decide to use it in execstack :-)
 may be next bintuils will make it public.
  -Alejandro

--- ./src/prelink.h.orig	2003-12-09 22:59:17.000000000 -0300
+++ ./src/prelink.h	2003-12-09 23:04:03.000000000 -0300
@@ -44,6 +44,8 @@
 #define SHT_GNU_LIBLIST		0x6ffffff7
 #endif
 
+#define PT_GNU_STACK		(PT_LOOS + 0x474e551)
+
 struct prelink_entry;
 struct prelink_info;
 struct PLArch;
